lib/better_words/schema.ex

```diff
@@ -0,0 +1,16 @@
+defmodule BetterWords.Schema do
```

Comment:
This file exists to give us our common settings across any schema files we create. Check out lib/better_words/words/worst_word.ex in this PR to see it in action.